src/components/CodeView/index.tsx

```diff
@@ -25,7 +25,7 @@ import GlobalLinterMessages from '../GlobalLinterMessages';
 import SlowPageAlert from '../SlowPageAlert';
 
 // This is how many lines of code it takes to slow down the UI.
-const SLOW_LOADING_LINE_COUNT = 500;
+const SLOW_LOADING_LINE_COUNT = 1000;
```

Comment:
Just wondering, are we just taking arbitrary guesses at this number? Might a file with 500 long lines be an issue, but one with 600 short lines not be an issue? I think this is fine to do, but I'm wondering where we are getting these numbers.

Reply:
Yeah, just a guess. I tested the files in https://github.com/mozilla/addons-code-manager/issues/947 which are legitimately slow and they are still flagged as such.

Reply:
Yes, probably something for a follow-up, but just the line count might not be a good metric. Minified files only have a few lines, but they are often very slow to load.

Reply:
@wagnerand can you send me examples of this? When I profiled the browse view I could did not find a noticeable difference in minified files, only in diff view (which is mitigated somewhat). If necessary, we could change the detection to *waves math hands* some calculation of line width and line count.

Reply:
Will do next time I come across it!

Reply:
We could maybe use the number of characters instead, line width/count wouldn't matter anymore.

Reply:
But where's the fun in a simple solution like that? 😉

Reply:
I'm not sure if character count would be enough to detect how a single long line of minified code causes performance problems. It's worth a try, though.

Reply:
A single metric, whatever it is, might not be enough.